subtests/docker_cli/events/test_event_parsing.py

```diff
@@ -63,7 +63,7 @@ def test_1_10(self):
              "2016-04-06T09:54:07.108386649-04:00",
              "2016-04-06T09:54:07.442663135-04:00",
              "2016-04-06T09:54:07.846336619-04:00",
-             "2016-04-06T09:55:00.000000000-04:00"]
+             "2016-04-06T13:55:00.000000000Z"]
         cid = ("ae5f8ee3cdc14512fbc6f908f06cc859"
                "0b53d0eb9e36cf543165d130f1169a0e")
```

Comment:
I know it's out of scope, but while you're changing this file, should the ``cid`` test include an item for ``sha256:blah``?